src/add-simple-form-value.js

```diff
@@ -21,5 +23,27 @@ export function addSimpleFormValue(value, options) {
     triplesToAdd.push(newTriple);
   }
 
+  // Add missing FormDataNode types for subjects without (rdf:type form:FormDataNode)
+  const triplesToAddLength = triplesToAdd.length;
+
+  for (let index = 0; index < triplesToAddLength; index++) {
+    const triple = triplesToAdd[index];
+
+    if (
+      triple.subject?.value?.startsWith(
+        "http://data.lblod.info/form-data/nodes/"
```

Comment:
You're depending here on the structure of a URI, which is considered bad practice. (In a certain future, we should be able to define the base-uri)
Anyway, i am unsure why `triplesForPath` is not updated; this is way closer to the source?

Reply:
Updated the string to URI_TEMPLATE var which is also used by the function that generates them. Will move code to triplesForPath

Reply:
This is the only way I found it working after searching for many different ways closer to "source" both in the triplesForPath and triples-for-unscoped-path where they get added. Could you tell me what is "broken/wrong" with this code? I tested it thoroughly and it seems to be doing what it should, this is blocking us a lot and have spend a lot of time looking for this solution.